### DISCUSSION TITLE
[Russian] Artifact spelling

### DISCUSSION THREAD

**Discussion by a1exsh** [ihhub/fheroes2#6020] [General]
(text below in Russian, as it refers to language-specific grammar rules)

@Branikolog приветствую! :)

Всё-таки мне не даёт покоя правописание артефактов...  В очередной раз полистав справочники, я склоняюсь к мысли, что правильно было бы следовать написанию, при котором лишь первая буква даётся заглавной, а именно: "Книга стихий", "Символ жизни", и т.п.

Основанием для такого решения предлагаю считать следующий параграф из справочника Розенталя:

> **§ 23. Названия документов, памятников старины, произведений искусства**
> 
> С прописной буквы пишутся первое слово и собственные имена в названиях важнейших документов, государственных законов, памятников старины, предметов и произведений искусства, например: Конституция Российской Федерации, Федеративный договор, Меморандум об экономической политике, Эрфуртская программа, Великая хартия вольности, Атлантическая хартия, Пакт Мира (с особым стилистическим употреблением второго слова), Декларация независимости, Договор о дружбе, сотрудничестве и взаимной помощи (предшествующее имя прилагательное, образованное от названий стран, заключивших договор, пишется со строчной буквы, например: …российско-китайский Договор о…), Лаврентьевская летопись, Венера Милосская, Исаакиевский собор (но: собор Парижской Богоматери – первое слово обозначает родовое понятие), храм Василия Блаженного (то же), здания Сената и Синода (в Петербурге), Сан Марко, Сент Антонио (в названиях церквей и соборов элементы Сан, Сент и т. п. пишутся без дефиса, в отличие от географических названий, [см. § 17, п.6](http://www.rosental-book.ru/ortho_iii.html#sect17.6)), Девятая симфония Бетховена, Первая баллада Шопена. Однако в настоящее время закрепились написания царь-колокол, царь-пушка (со строчной буквы).

http://www.rosental-book.ru/ortho_iii.html#sect23

В то же время, некоторые артефакты попадают в отдельную категорию из соседнего параграфа 25 "Названия орденов, медалей, знаков отличия, премий": http://www.rosental-book.ru/ortho_iii.html#sect25
Соответственно, получаем `медаль "За отвагу"`, и т.п.

Что скажете?

В любом случае, дабы не обсуждать этот вопрос по десятому разу в каждом новом улучшении перевода, предлагаю выбранное решение закрепить таким образом, чтобы последующие переводчики обращали на это внимание. ;)

**Comment by Branikolog** [comment]
@a1exsh 
Доброго дня.
Да, насчет медалей я находил эту информацию. :) И насчет в целом написания с первой заглавной я тоже согласен, т.к. сам несколько часов изучал правила и тоже находил примеры, очень близкие к нашей ситуации именно с первой заглавной (Ну и имена собственные, конечно, если присутствуют).
Но в защиту текущей реализации могу тоже привести несколько доводов: Во-первых, имея одинаковое написание везде - намного проще это всё в коде реализовать. Не нужно для каждого отдельного предмета указывать его способ написания. К тому же, я не уверен, что наш код позволяет писать названия предметов не с заглавной (если мы говорим о тех же медалях), если название артефакта встречается где-то в середине сгенерированного текста. И в целом, написание со всех заглавных намного лучше читается по тексту, в плане заметности. Игроку намного проще в тексте глазами зацепиться за название, нежели оно будет написано лишь с одной заглавной, пускай это даже и не совсем по правилам. Но в целом, если мы сможем избежать проблем с написанием по коду, я абсолютно не против. Но это лучше делать отдельным PR, чтобы можно было посмотреть не всплывут ли еще какие проблемы.

**Comment by a1exsh** [comment]
> если мы сможем избежать проблем с написанием по коду, я абсолютно не против. Но это лучше делать отдельным PR

ОК, тогда посмотрим после того, как закроем текущий ПР с улучшениями :)

**Reply by Branikolog** [reply]
Я рассчитываю на ваши "связи" с опытными носителями языка. :)

**Reply by a1exsh** [reply]
А я уже как раз привлёк их к процессу ;)
